_docs/extensions/generate.py

```diff
@@ -175,22 +198,22 @@ def key(row: tuple[str, ...]) -> tuple[str, ...]:
         status, _name, _kind, desc = row
         return status, desc
 
-    rows = [(_status(e), _name(e), _kind(e), _rel_description(e)) for e in entries]
-    return _LIBS_TABLE_HEADER + _rst_rows(sorted(rows, key=key))
+    rows = [
+        (_status(entry), _name(entry), _kind(entry), _rel_description(entry))
+        for entry in filter(_not_unlisted, entries)
```

Comment:
It seems a bit odd to use a `filter()` inside of a list comprehension. Any reason to do this over something like this?

```suggestion
        for entry in (row for row in entries if _not_unlisted(row))
```

Reply:
Good point, instead of a nested list comprehension I've added a conditional to the outer list comprehension, thanks.